includes/functions-install.php

```diff
@@ -22,6 +22,14 @@ function yourls_check_database_version() {
 /**
  * Get DB version
  *
+ * The regex removes everything that's not a number at the start of the string, or remove anything that's not a number and what
+ * follows after that.
+ *   'omgmysql-5.5-ubuntu-4.20' => '5.5'
+ *   'mysql5.5-ubuntu-4.20'     => '5.5'
+ *   '5.5-ubuntu-4.20'          => '5.5'
+ *   '5.5-beta2'                => '5.5'
+ *   '5.5'                      => '5.5'
+ *
```

Comment:
There is no regex in this function. Why document this here instead of on `yourls_sanitize_version()`? (Which already has a variant of this in its docstring.)

https://github.com/YOURLS/YOURLS/blob/4340a65b9af050c4ebf9cc9d17ef598dc6bdb03a/includes/functions-formatting.php#L232-L243

```suggestion
```